kodilla-code-coverage/src/test/java/com/kodilla/hibernate/invoice/repository/InvoiceRepositoryTest.java

```diff
@@ -49,8 +59,14 @@ public void testInvoiceRepositorySave() {
 
         //Then
         Assert.assertNotEquals(0, id);
+        Assert.assertTrue(invoice.getItems().contains(item));
+        Assert.assertTrue(invoice.getItems().contains(item2));
+        Assert.assertTrue(invoice.getItems().contains(item3));
+        Assert.assertEquals(itemRepository.count(), 3);
```

Comment:
Asercje powinny sprawdzac czy wszystkie Item'y zostaly zapisane do bazy danych. Mozesz np zapisac 
Assert.assertNotEquals(0, invoice.getItems().get(0).getId());
Assert.assertNotEquals(0, invoice.getItems().get(1).getId());
itd
...
Obecna wersja nie sprawdza czy itemy sie zapisaly bo 'invoice' to jest obiektem ktory posluzyl do zapisania wartosci w bazie